releasenotes/notes/add-python311-support-3a939c4697ecd896.yaml

```diff
@@ -0,0 +1,5 @@
+---
+upgrade:
+  - |
+    Added support for running with Python 3.11.
+    At the the time of the release, Psi4, PyQuante2 and Sparse didn't have a python 3.11 version.
```

Comment:
Since PyQuante2 driver is part of deprecated code, and we do not have any newer driver based on it goings forwards, should we even mention it here. Maybe this should be a new feature section too rather than upgrade - I think upgrade is more about things the user should do to upgrade their code - e.g. the removal of deprecated code means they now have to upgrade it if they did not before.